### ISSUE TITLE
[FR] x-alarms ServiceName - Dimension

### ISSUE THREAD

**Issue by akanungo22** [compose-x/ecs_composex#634]
In `x-alarms`, when have a way to automatically set the Dimensions for a service (cluster and service name).

**Comment by JohnPreston** [comment]
```yaml
services:
  my-app: {}
  
x-alarms:
  high-cpu:
    Properties:
    Dimensions: service::my-app
```

or

```yaml
services:
  my-app:
  x-alarms:
    high-cpu:
      Properties: {} # Dimensions etc. replaced with the service values
```


would resolve to

```yaml
services:
  my-app: {}
  
x-alarms:
  high-cpu:
    Properties:
    Dimensions:
      - Name: clusterName
        Value: <the cluster name>
      - ServiceName:
        Value: <the ECS Service ID> 
```

**Comment by akanungo22** [comment]
services:
  my-app: {}
  
x-alarms:
  high-cpu:
    Properties:
    Dimensions:
      - Name: ClusterName
        Value: <the cluster name>
      - ServiceName:
        Value: <the ECS Service ID> 


Is this service ID or the service name?

**Comment by JohnPreston** [comment]
Implemented & will be released officially (and documented) with 0.23

```yaml
x-alarms:
  service-alarm:
    Properties:
      ActionsEnabled: true
      AlarmDescription: A simple CW alarm
      ComparisonOperator: GreaterThanOrEqualToThreshold
      DatapointsToAlarm: 1
      EvaluationPeriods: 5
      MetricName: TotalLagForTopicAndConsumerGroup
      Namespace: lag-metrics-v4
      Period: 60
      Statistic: Sum
      Threshold: 1.0
      TreatMissingData: notBreaching
    MacroParameters:
      ServiceName: bignicefamily
```

**Comment by JohnPreston** [comment]
Fixed in #650
